### PR TITLE
kubevirtci, presubmit: Introduce k8s-1.25-ipv6

### PR DESCRIPTION
**Pull request by oshoval**
Depends on https://github.com/kubevirt/kubevirtci/pull/912

### CODE REVIEW

**Comment by oshoval** [comment]
```
error: resource mapping not found for name: "calico-kube-controllers" namespace: "kube-system" from "/provision/cni.yaml": no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
```
Need to bump calico or so

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml
Comment (dhiller) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml:532]: Let's start with `always_run: false` and `optional: true`.

**Review by dhiller — CHANGES_REQUESTED** [review]

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml
Comment (oshoval) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml:532]: np, it was still draft ;]

**Comment by oshoval** [comment]
Addressed comments,
Removed the manual apply of https://github.com/kubevirt/kubevirtci/pull/912

/hold
until https://github.com/kubevirt/kubevirtci/pull/912 is merged

**Review by dhiller — APPROVED** [review]
/approve

Since the new lane is triggered manually, you could also merge it right away, but I'll leave that up to you :)

**Comment by oshoval** [comment]
Right thanks
lets merge it

/hold cancel

**Pull request merged**
